ufo2fontir/src/toir.rs

```diff
@@ -252,4 +252,53 @@ mod tests {
         .unwrap();
         assert_eq!(HashSet::from([0x007C]), glyph.codepoints);
     }
+
+    #[test]
+    pub fn component_transforms() {
+        let mut c = Component::new(
+            Name::new("A").unwrap(),
+            AffineTransform {
+                x_scale: 1.0,
+                xy_scale: 0.0,
+                yx_scale: 0.0,
+                y_scale: 1.0,
+                x_offset: 0.0,
+                y_offset: 0.0,
+            },
+            None,
+            None,
+        );
+        assert_eq!(
+            to_ir_component(&c).transform,
+            Affine::new([1.0, 0.0, 0.0, 1.0, 0.0, 0.0])
+        );
+
+        c.transform = AffineTransform {
+            x_scale: 1.0,
+            xy_scale: 0.0,
+            yx_scale: 0.0,
+            y_scale: 1.0,
+            x_offset: 10.0,
+            y_offset: 10.0,
+        };
+        assert_eq!(
+            to_ir_component(&c).transform,
+            Affine::new([1.0, 0.0, 0.0, 1.0, 10.0, 10.0])
+        );
+
+        // <component base="a" xScale="0.4366" xyScale="-0.4366" yScale="0.4415" yxScale="0.4415" xOffset="282" yOffset="5" identifier="5402E799"/>
+        c.transform = AffineTransform {
+            x_scale: 0.4366,
+            xy_scale: -0.4366,
+            y_scale: 0.4415,
+            yx_scale: 0.4415,
```

Comment:
I think it might help avoid any further confusions if you keep the struct fields here ordered the same way they are defined in the norad::AffineTransform struct, so 
```suggestion
            x_scale: 0.4366,
            xy_scale: -0.4366,
            yx_scale: 0.4415,
            y_scale: 0.4415,
```